neutrino.go

```diff
@@ -733,6 +734,9 @@ func NewChainService(cfg Config) (*ChainService, error) {
 		})
 	}
 
+	s.utxoScanner = NewUtxoScanner(&s)
+	s.utxoScanner.Start()
```

Comment:
Since this starts a goroutine, I guess this should be called in `ChainService.Start` so that a waitgroup item can be added to make sure the scanner is finished when the service is shutting down.